ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-ref.yaml

```diff
@@ -11,7 +11,7 @@ ref:
     mount_path: /secret
   env:
   - name: E2E_VERSION
-    default: "v2.1.4"
+    default: "kube-burner-ocp"
```

Comment:
New E2E is released and this branch got deleted. We need to use v2.1.5 E2E_VERSION

Reply:
Ack, updating this.